index.html

```diff
@@ -24,8 +24,9 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
     </li>
   </ul>
 

```

Comment:
create another list as in lines 14 - 25: https://github.com/xiomaraquinonez/todolist/pull/1/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR14-R25

but this time looping over completedTodos and activeTodos:

`<li v-for="todo in todos" :key="todo.id" class="flex">`  ->  `<li v-for="todo in activeTodos" :key="todo.id" class="flex">`

`<li v-for="todo in todos" :key="todo.id" class="flex">`  ->  `<li v-for="todo in completedTodos" :key="todo.id" class="flex">`